src/uct/ib/mlx5/dv/ib_mlx5_ifc.h

```diff
@@ -202,7 +202,10 @@ struct uct_ib_mlx5_cmd_hca_cap_bits {
     uint8_t    pps[0x1];
     uint8_t    pps_modify[0x1];
     uint8_t    log_max_msg[0x5];
-    uint8_t    reserved_at_1c8[0x4];
+    uint8_t    multi_path_xrc_rdma[0x1];
+    uint8_t    multi_path_dc_rdma[0x1];
+    uint8_t    multi_path_rc_rdma[0x1];
```

Comment:
in latest PRM it is renamed to something like `dp_ordering_ooo_rw_rc`

Reply:
used latest PRM and fixed all names, thanks for noticing.